code/modules/keybindings/bindings_client.dm

```diff
@@ -5,7 +5,7 @@
 	set hidden = TRUE
 
 	keys_held[_key] = world.time
-	var/movement = SSinput.movement_keys[_key]
+	var/movement = prefs.movement_keys[_key] // yogs - Custom keybinds
```

Comment:
I thought we were rebasing why is it called yogs keybinds not tg keybinds lol??????!?